src/components/PrintModal/PrintModal.js

```diff
@@ -497,7 +501,10 @@ class PrintModal extends React.PureComponent {
                 dataElement="printModalCloseButton"
                 title="action.close"
                 img="ic_close_black_24px"
-                onClick={this.closePrintModal}
+                onClick={() => {
+                  this.cancelPrint();
```

Comment:
If we call `this.cancelPrint` here then we also need to change `line 495` to call `cancelPrint` as well to make the behavior consistent.

Reply:
done